packages/ti_mandiant_advantage/data_stream/threat_intelligence/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,399 @@
+---
+description: Pipeline for processing Mandiant IOCs.
+processors:
+- rename:
+      field: message
+      target_field: event.original
+      ignore_missing: true
```

```suggestion
- rename:
      field: message
      target_field: event.original
      ignore_missing: true
      if: 'ctx.event?.original == null'
```

This if check just helps us prevent conflict in the event the document already has the field set ie when sending from Logstash